timeseries/src/autogluon/timeseries/models/ensemble/array_based/models.py

```diff
@@ -0,0 +1,7 @@
+from .abstract import ArrayBasedTimeSeriesEnsembleModel
+from .regressor import SimpleAverageEnsembleRegressor
+
+
+class ArrayBasedSimpleAverageEnsemble(ArrayBasedTimeSeriesEnsembleModel):
+    _default_model_name = "ArrayBasedSimpleAverageEnsemble"
+    _regressor_type = SimpleAverageEnsembleRegressor
```

Comment:
Nit: Since we already have `SimpleAverageEnsemble`, maybe we can use `SimpleMedianEnsemble` as a very simple representative of the array-based ensembles?

Reply:
nice!